service/history/historyEngine.go

```diff
@@ -1155,6 +1155,14 @@ func (e *historyEngineImpl) DescribeWorkflowExecution(
 		result.WorkflowExecutionInfo.CloseTime = common.Int64Ptr(completionEvent.GetTimestamp())
 	}
 
+	switch executionInfo.State {
+	case persistence.WorkflowStateRunning,
```

Comment:
should WorkflowStateCreated alsos consider as open?

Reply:
I don't know about that. @wxing1292, @longquanzheng

Reply:
+1 created should be open as well

Reply:
Oh your code is correct. @yux do you have concern about considering created as open? From customer's perspective they need it open to make sure the workflow is not closed. 

On the other hand, I think it will be useful if we could expose the close statuses: terminated/failed/canceled/timeouted. I think we can just do it together.

Reply:
I think we are exposing the close statuses if exectutionInfo.State is WorkflowStateCompleted.

Reply:
@longquanzheng  Close status is already exposed. But we don't have a close status for open workflows so that field will be empty in that case. I think the point of this PR is to add a new field so that user doesn't need to determine if the workflow is running or not by checking if the closeStatus is nil.  

So I think the value for `isWorkflowRunning` can simply determined by `executionInfo.State == persistence.WorkflowStateCompleted`

Reply:
Missed the next line. It makes sense that WorkflowStateCreated is workflow open.